src/main/resources/db/migration/h2/V1_1__create_audit_event_table.sql

```diff
@@ -1,10 +1,30 @@
-create table audit_event
+CREATE SCHEMA IF NOT EXISTS staff;
```

Comment:
As this create two new schemas all the existing audit data will be left in `public`. I guess there is nothing to stop us coping the data into the new staff schema thought

Reply:
I would potentially update the existing migrations to be staff and then add additional migration to create the prisoner schema and tables just so it shows the evolution of the db e.g. V3_0__add_prisoner_audit_schema.sql`.